project_report/templates/report_base.html

```diff
@@ -4,72 +4,123 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" crossorigin="anonymous">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" crossorigin="anonymous">
     <style type="text/css">
-        #edgen_logo { width: 150px; }
-        h1 { font-size: 2.7em; }
-        h2 { font-size: 2.0em; }
-        h3 { font-size: 1.3em; }
-        .pagebreak { page-break-before: always; }
+        #edgen_logo { width: 70px; margin-left: auto; margin-right: auto; display: inline-block;; }
+
+        h1 { font-size: 1.4em; font-family:Avantgarde; sans-serif; line-height: 70%; padding-top: 80px; }
+        h2 { font-size: 1.7em; line-height: 160%; font-family:Avantgarde, sans-serif; }
+        h3 { font-size: 1.3em; line-height: 160%; font-family:Avantgarde; sans-serif; }
+        h4 { font-size: 2.7em; line-height: 300%; font-family:Avantgarde; sans-serif; }
+        h5 { font-size: 1.1em; line-height: 140%; font-family:Avantgarde; sans-serif; }
+
+        div.charts {
+           display: block;
+           margin-left: auto;
+           margin-top: auto; }
+
+        table {
+            font-family: arial, sans-serif;
+            width: 100%;
+            text-align: left;
+        }
+        td, th {
+            border: none;
+            text-align: left;
+            padding: 4px;
+            font-size: 1.2em;
+        }
+
+        @page {
+        margin-top: 2cm;
+        margin-bottom: 2cm;
+        }
+
+        div.page-footer { margin-bottom: 5cm; }
+        @media screen {
+          div.divFooter {
+           display: none;
+          }
+        }
+        @media print {
+          div.divFooter {
+            position: fixed;
+            bottom: 0%;
+            left: 40%;
+          }
+        }
     </style>
     <meta charset="UTF-8">
-    {% block head %}
-    <title>Summary for project {{ project_name }}</title>
-    {% endblock %}
+
 </head>
+<div class="divFooter"><img id="edgen_logo" src="http://s29.postimg.org/8qp6u1agn/EG_logo_final_version_colourwhite_cropped.png"/></div>
+    <div id="page-header" style="text-align: center;">
```

Comment:
The css here should be moved to the general style section above and given a class name